kong/plugins/grpc-web/handler.lua

```diff
@@ -51,7 +50,7 @@ function grpc_web:access(conf)
   end
 
   local dec, err = deco.new(
-    kong_request_get_header("Content-Type"),
+    ngx_var.http_content_type,
```

Comment:
It may not correct, since now nginx could return multiple headers in one value, like "a, b, c".
could you check other places? thanks.

Reply:
@bungle , could you verify it? thanks.